<launch>
	<include file="$(find rbe_500)/launch/final_project_part_2.launch"/>
	<node pkg="rbe_500" type="switch_controller_server.py" name="switch_controller_node"/>

</launch>
